packages/frontend/.ember-cli.js

```diff
@@ -12,5 +12,5 @@ module.exports = {
   disableAnalytics: false,
   liveReload: true,
   host: '0.0.0.0',
-  usePods: true,
+  componentStructure: 'nested',
```

Comment:
@NullVoxPopuli what's this?

Reply:
defaults the component blueprint generator to do:

`app/components/{component-name}/index.{js,hbs}`

Reply:
Instead of `app/components/{component-name}.{js,hbs}`?

Reply:
yup!